.kitchen.yml

```diff
@@ -7,7 +7,9 @@ driver:
     # TODO: make the port an attribute in the test cookbook so we can enable some
     # concurrency.
     - ["forwarded_port", {guest: 9292, host: 9292}]
-
+  synced_folders:
+    - ['.', '/vagrant/code']
```

Comment:
As this is only required for the windows tests, should we remove it from the main config?